### ISSUE TITLE
[Input Time Picker] Provide a valid label (CAL-AMP25) 

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#13533]
**monday.com sync:** #10822034929

### Violation: 
 Provide a valid label for form fields 

### WCAG Reference: 
- [2.5.3 Label in Name](https://www.w3.org/WAI/WCAG22/Understanding/label-in-name) 

### Recommended solution
_Some investigation_ is needed to determine if we can add context to the component on initial focus and then provide context to the component's values and context.

It looks like the component's context was removed when blurring was added to provide context to the HH / MM / SS, where the overall component's context was lost.

### Severity: 
 Critical 

### Media Type: 
 ARIA 

--- 
### Areas for Remediation: 
- Title: CAL-AMP25 Provide a valid label for form fields 
- Module: M4 Form Elements_Employee Dashboard 
- Finding ID: [CAL-AMP25](https://esri.hub.essentia11y.com/short-link/UFhDG6f3FqOc3V6c) 

**Issue**
There are form controls with visual labels that are not programmatically associated with the control. Examples include:
- Earliest offering
- Latest offering.

**User Impact**
Screen reader users will have difficulty determining the purpose of these controls. Speech input users will have difficulty navigating to them.

**Code Reference** 
 ```
<calcite-input-time-picker id="earliest-offering" name="earliest-offering" value="07:00" min="07:00" max="16:00" hour-format="user" placement="auto" scale="m" status="idle" step="60" calcite-hydrated="">(...)</calcite-input-time-picker>

```

**Steps to Reproduce**
1. Locate the form field.
2. Inspect it with Chrome DevTools.
3. In the Accessibility tab, expand the Computed Properties section.
4. Review the value for "Name".
 
**Recommendation**
Provide a valid label for form fields. The best way to do this is to use a ```<label>``` element with a for attribute set to the ID of the field it labels. For custom controls, use aria-labelledby on the control set to the ID of the labelling text.

**Compliant Code Example**
```
<!--Example using label element--> 
<label for="amount ">Amount:</label> <input id="Amount" type="text" name="amount" size="30"> 
---
<!--Example label has no "for" attribute but does wrap the input element)--> 
<label>Amount: <input id="amount " type="text" name="amount " size="30"></label> 
---
<!--Example using aria-labelledby--> 
<label id="lblAmount ">Amount:</label> <input aria-labelledby="lblAmount" id="Amount" type="text" name="amount" size="30"> 
---
<!-- Example using aria-label attribute - visual label exists but is not associated--> 
<span>Amount:</span> 
<input aria-label="Amount:" type="text" id="amount" size="30"> 
---
<!--Example using title attribute - visual label exists but is not associated--> 
<span>Amount:</span> 
<input title="Amount:" type="text" id="amount" size="30">
```

 **Issue Image** 
 https://desktopqa/aic_lap/Calcite/Calcite2025/CAL-AMP25.png 
 

 --- 
### Additional Resources: 
- [LA Platform Report Link](https://esri.hub.essentia11y.com/short-link/UFhDG6f3FqOc3V6c) 
- [Esri Accessibility Resources](https://esriis.sharepoint.com/sites/Accessibility)
- Report Source: Product accessibility evaluation conducted on Calcite by Level Access and delivered August, 2025.
